ste/sender-pageBlob.go

```diff
@@ -53,6 +53,12 @@ type pageBlobSenderBase struct {
 	// Using a automatic pacer here lets us find the right rate for this particular page blob, at which
 	// we won't be trying to move the faster than the Service wants us to.
 	filePacer autopacer
+
+	// destPageRangeOptimizer is necessary for managed disk imports,
+	// as it helps us identify where we actually need to write all zeroes to.
+	// Previously, if a page prefetched all zeroes, we'd ignore it.
+	// This would cause corruption on managed disks.
+	destPageRangeOptimizer *pageRangeOptimizer
```

Comment:
Nice name. Can you please rename the other one (the one in sender-pageBlobFromURL) to sourcePageRangeOptimizer, so that it fits into the same naming scheme.